rabbitmq-setup.sh

```diff
@@ -0,0 +1,17 @@
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | sudo tee /etc/apt/sources.list.d/rabbitmq.list && sudo apt-get update && sudo apt-get install rabbitmq-server
+sudo apt-get install cmake libboost-dev openssl libssl-dev libblkid-dev e2fslibs-dev libboost-all-dev libaudit-dev
+cd /tmp/
+git clone https://github.com/alanxz/rabbitmq-c
```

Comment:
Before doing all this, I think we should put a check to see if the `.a` and `.so` files aren't already in `/usr/local/lib/`. If it is, we should put a message saying so and skip this.